### PR TITLE
programatically fix straight quotes/apostrophes

### PR DESCRIPTION
**Pull request by jaredh159**

### CODE REVIEW

File: journal/modernized/02-chapter-1.adoc [hunk 1]
```diff
@@ -910,9 +910,9 @@ these Friends and William Edmundson met with much personal abuse.--See Rutty's _
 In the year 1676, John Tiffin,
 accompanied John Burnyeat through Westmoreland and Yorkshire, to London;
 and in 1684, into the north of England and Scotland.
-See Whiting''s Memoirs.
+See Whiting'`'`s Memoirs.
```
Comment (Henderjay) [inline journal/modernized/02-chapter-1.adoc]: This was an error in document, but thought you might want to see how it rendered it. Looks like it took the two apostrophes and turned the both the wrong way.

**Review by Henderjay — COMMENTED** [review]
These all looked good. Although I didn't see many or any that had the same situation that Branson had, with the punctuation OUTSIDE of the single quote. There is one place I marked where there were two apostrophes and it rendered weird, but it was an error in the original.

**Review by Henderjay — COMMENTED** [review]
these all look good. Looked at a large number and saw no problems.

File: journal/modernized/02-chapter-1.adoc [hunk 2]
```diff
@@ -890,59 +890,59 @@ there were but very few of them in that nation; with these few,
...
 In the next year, John Tiffin, visited Ireland again, with James Lancaster.
 At a meeting held on the green at Lurgan, in the course of this visit,
-these Friends and William Edmundson met with much personal abuse.--See Rutty's _History._
+these Friends and William Edmundson met with much personal abuse.--See Rutty`'s _History._
 {footnote-paragraph-split}
 In the year 1676, John Tiffin,
 accompanied John Burnyeat through Westmoreland and Yorkshire, to London;
 and in 1684, into the north of England and Scotland.
-See Whiting''s Memoirs.
+See Whiting'`'`s Memoirs.
```
Comment (jaredh159) [inline journal/modernized/02-chapter-1.adoc]:
```suggestion
See Whiting'`s Memoirs.
```

**Comment by jaredh159** [comment]
sweet, thanks. i fixed the one double apostrophe you found, going to merge this one as well.

**Pull request merged**